cirq-google/cirq_google/devices/google_noise_properties_test.py

```diff
@@ -0,0 +1,258 @@
+    cirq.ISwapPowGate: 32.0,
+    cirq.CZPowGate: 32.0,
+    # cirq.WaitGate is a special case.
+}
+
+
+# These properties are for testing purposes only - they are not representative
+# of device behavior for any existing hardware.
+def sample_noise_properties(
+    system_qubits: List[cirq.Qid], qubit_pairs: List[Tuple[cirq.Qid, cirq.Qid]]
+):
+    # Known false positive: https://github.com/PyCQA/pylint/issues/5857
+    return GoogleNoiseProperties(  # pylint: disable=unexpected-keyword-arg
+        gate_times_ns=DEFAULT_GATE_NS,
+        t1_ns={q: 1e5 for q in system_qubits},
+        tphi_ns={q: 2e5 for q in system_qubits},
+        readout_errors={q: np.array([0.001, 0.01]) for q in system_qubits},
+        gate_pauli_errors={
+            **{OpIdentifier(g, q): 0.001 for g in SINGLE_QUBIT_GATES for q in system_qubits},
+            **{
+                OpIdentifier(g, q0, q1): 0.01
+                for g in SYMMETRIC_TWO_QUBIT_GATES
+                for q0, q1 in qubit_pairs
+            },
+        },
+        fsim_errors={
+            OpIdentifier(g, q0, q1): cirq.PhasedFSimGate(0.01, 0.03, 0.04, 0.05, 0.02)
+            for g in SYMMETRIC_TWO_QUBIT_GATES
+            for q0, q1 in qubit_pairs
+        },
+    )
+
+
+def test_zphase_gates():
+    q0 = cirq.LineQubit(0)
+    props = sample_noise_properties([q0], [])
+    model = NoiseModelFromGoogleNoiseProperties(props)
+    circuit = cirq.Circuit(cirq.Z(q0) ** 0.3)
+    noisy_circuit = circuit.with_noise(model)
+    assert noisy_circuit == circuit
+
+
+@pytest.mark.parametrize(
+    'op',
+    [
+        (cirq.Z(cirq.LineQubit(0)) ** 0.3).with_tags(cirq_google.PhysicalZTag),
+        cirq.PhasedXZGate(x_exponent=0.8, z_exponent=0.2, axis_phase_exponent=0.1).on(
+            cirq.LineQubit(0)
+        ),
+    ],
+)
+def test_single_qubit_gates(op):
+    q0 = cirq.LineQubit(0)
+    props = sample_noise_properties([q0], [])
+    model = NoiseModelFromGoogleNoiseProperties(props)
+    circuit = cirq.Circuit(op)
+    noisy_circuit = circuit.with_noise(model)
+    assert len(noisy_circuit.moments) == 3
+    assert len(noisy_circuit.moments[0].operations) == 1
+    assert noisy_circuit.moments[0].operations[0] == op.with_tags(PHYSICAL_GATE_TAG)
+
+    # Depolarizing noise
+    assert len(noisy_circuit.moments[1].operations) == 1
+    depol_op = noisy_circuit.moments[1].operations[0]
+    assert isinstance(depol_op.gate, cirq.DepolarizingChannel)
+    assert np.isclose(depol_op.gate.p, 0.00081252)
+
+    # Thermal noise
+    assert len(noisy_circuit.moments[2].operations) == 1
+    thermal_op = noisy_circuit.moments[2].operations[0]
+    assert isinstance(thermal_op.gate, cirq.KrausChannel)
+    thermal_choi = cirq.kraus_to_choi(cirq.kraus(thermal_op))
+    assert np.allclose(
+        thermal_choi,
+        [
+            [1, 0, 0, 9.99750031e-01],
+            [0, 2.49968753e-04, 0, 0],
+            [0, 0, 0, 0],
+            [9.99750031e-01, 0, 0, 9.99750031e-01],
+        ],
+    )
+
```

Comment:
one nice check here would be:
compute the pauli error for the two noise channels. Add it up, and compare to the value in `gate_pauli_errors`

Reply:
Done.